.github/workflows/static_analysis.yml

```diff
@@ -20,17 +20,13 @@ jobs:
     - name: Cleanup repository
       run: rm -rf vendor/
 
-    - name: Vet
-      run: go vet -tags ci ./...
-
-    - name: Goimports
-      run: test -z $(goimports -e -d . | tee /dev/stderr)
-
-    - name: Gocyclo
-      run: gocyclo -over 50 .
-
-    - name: Golint
-      run: golint -set_exit_status $(go list -tags ci ./...)
+    - name: golangci-lint
+      uses: golangci/golangci-lint-action@v2.4.0
+      with:
+        version: v1.37
+        skip-go-installation: true
+        # skip-pkg-cache: true
+        # skip-build-cache: true
```

Comment:
Not sure if we really need to keep these. We can always add them in if needed sometime in the future.